settings.gradle

```diff
@@ -2,7 +2,11 @@ pluginManagement {
     repositories {
         gradlePluginPortal()
         maven {
-			url 'https://gitlab.com/api/v4/projects/26758973/packages/maven'
-		}
+            url 'https://gitlab.com/api/v4/projects/26758973/packages/maven'
```

Comment:
This should be labeled (presumably with the name field) as FancyGradle for future reference.